test/uk/gov/hmrc/merchandiseinbaggage/smoketests/ImportJourneySpec.scala

```diff
@@ -20,10 +20,12 @@ import uk.gov.hmrc.merchandiseinbaggage.model.api.YesNo.{No, Yes}
 import uk.gov.hmrc.merchandiseinbaggage.model.api.{Email, Name}
 import uk.gov.hmrc.merchandiseinbaggage.model.core.PurchaseDetailsInput
 import uk.gov.hmrc.merchandiseinbaggage.smoketests.pages._
-import uk.gov.hmrc.merchandiseinbaggage.stubs.MibBackendStub._
+import uk.gov.hmrc.merchandiseinbaggage.stubs.MibBackendStub
 
 class ImportJourneySpec extends BaseUiSpec {
 
+  private val stub = injector.instanceOf[MibBackendStub]
```

Comment:
Once the changes in the `MibBackendStub` have been done, please revert these changes, thanks